components/ide/code/gitpod-web/BUILD.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+
+packages:
```

Comment:
Do we need this component? I hoped we can remove eventually all code related component out of this repo and only configure installer in code.

Can we publish a ready image already?

Reply:
@mustard-mh will check tomorrow morning about publishing to gpc registry

Reply:
https://github.com/gitpod-io/gitpod/pull/16073#issuecomment-1410099081